Mathlib/CategoryTheory/Limits/FunctorCategory/Shapes/Images.lean

```diff
@@ -0,0 +1,73 @@
+/-
+Copyright (c) 2025 Dagur Asgeirsson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Dagur Asgeirsson
+-/
+module
+
+public import Mathlib.CategoryTheory.Limits.Shapes.Images
+public import Mathlib.CategoryTheory.Limits.FunctorCategory.EpiMono
+public import Mathlib.Tactic.CategoryTheory.CategoryStar
+
+/-!
+
+# The category of type-valued functors has images
+-/
+
+@[expose] public section
+
+universe u
+
+namespace CategoryTheory
+
+open Limits
+
+variable {C : Type*} [Category* C]
+
+/-- The image of a natural transformation between type-valued functors -/
+@[simps obj map]
+def NatTrans.image {F G : C ⥤ Type u} (f : F ⟶ G) : C ⥤ Type u where
```

Comment:
Could you consider using `CategoryTheory.Subpresheaf.Image`?
(This is for functors `Cᵒᵖ ⥤ Type u`, but we may also deduce the `HasImages` instances via an equivalence of categories.)

Reply:
(Otherwise, generalizing `Subpresheaf` by removing the `ᵒᵖ` in the definitions could be a good idea.)

Reply:
This is a little extra work! But if you are happy doing it, you may certainly do so! (In which case, we may use the name `Subfunctor`)

Reply:
Here is the generalization/renaming PR: #32746